src/Classes/ForbiddenClassNameExtension.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Classes;
+
+interface ForbiddenClassNameExtension
```

Comment:
Please put `@api` here and explanation what it is for and how to register it. Example: https://apiref.phpstan.org/1.11.x/source-src.PhpDoc.StubFilesExtension.html#5